resources/views/includes/field-label.blade.php

```diff
@@ -1,10 +1,6 @@
 <label for="formProperties-{{ $key }}" class="lfb-label">
     {!! $label !!}
-    @if ((!isset($mode) || (isset($mode) and $mode != 'view')) and isset($rules)
-        and array_key_exists('formProperties.' .  $key, $rules) && str_contains($rules['formProperties.' .  $key], 'required'))
+    @if ((!isset($mode) || (isset($mode) and $mode != 'view')) && $this->isFieldRequired($key))
         <sup>*</sup>
     @endif
-    @if(isset($tooltip) && $tooltip)
```

Comment:
Is it intended to remove the tooltip section here? I don't see that it was added somewhere else and this seems to break the tooltip functionality.

Reply:
Thanks, It was accidentally deleted. I will add it back.